src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -2152,6 +2182,12 @@ public Authentication getDefaultAuthentication() {
         return ACL.SYSTEM;
     }
 
+    /* TODO 1.592+
+    @Override public Authentication getDefaultAuthentication(Queue.Item item) {
```

Comment:
As plugin maintainer I decree that all annotations on a method shall be on lines preceding the `(public|protected|/*package*/|private)`